examples/explore_browser_use/planner.py

```
@@ -0,0 +1,57 @@
+from browser_use.llm import ChatOpenAI
+from browser_use import Agent, BrowserSession
+from dotenv import load_dotenv
+from PIL import Image, ImageFont
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+load_dotenv()
+
+import asyncio
```

Ruff (0.11.9)

Module level import not at top of file

(E402)